text/closest_string.ts

```diff
@@ -20,19 +25,14 @@ const getWordDistance = levenshteinDistance;
  * ```
  *
  * @param givenWord The string to measure distance against
- * @param possibleWords The string-array that will be sorted
+ * @param possibleWords The string-array to pick the closest string from
  * @param options An options bag containing a `caseSensitive` flag indicating
  * whether the distance should include case. Default is false.
- * @returns A sorted copy of possibleWords
- * @note
- * the ordering of words may change with version-updates
- * e.g. word-distance metric may change (improve)
- * use a named-distance (e.g. levenshteinDistance) to
- * guarantee a particular ordering
```

Comment:
I see `@note` not rendered at all https://jsr.io/@std/text@1.0.0-rc.2/doc/~/closestString